packages/google_maps_flutter/lib/src/google_map.dart

```diff
@@ -34,6 +34,7 @@ class GoogleMap extends StatefulWidget {
     this.polygons,
     this.polylines,
     this.circles,
+    this.padding = const EdgeInsets.all(0),
```

Comment:
add documentation above explaining this.

Reply:
Added `If no padding is specified default padding will be 0.`